ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1082,7 +1082,10 @@ class NavigationBindings {
         }
 
         objectEach(options, (option, key): void => {
-            if (key === 'typeOptions') {
+            if (
+                key === 'typeOptions' &&
+                visualOptions['type'] !== 'basicAnnotation' // #23575
```

Comment:
Could you please reuse `annotationsNonEditable` variable, to keep all non editable fields in one place?

```
public static annotationsNonEditable = {
  ...
  basicAnnotation: ['typeOptions']
};
```
Then:
```
if (
  key === 'typeOptions' &&
  annotationsNonEditable[visualOptions.type].indexOf('typeOptions') === -1
}
```



It's better to keep those limitations centralized in one place rather than scattered throughout the code.

Reply:
Thanks for the explanation!